theseus/geometry/lie_group.py

```diff
@@ -44,6 +45,17 @@ def _init_data(*args: Any) -> torch.Tensor:
     def dof(self) -> int:
         pass
 
+    @staticmethod
+    @abc.abstractmethod
+    def rand(
+        *size: int,
```

Comment:
Is `size` meant for batch size? If so we should rename and put `1` as default.

Reply:
`size` should be a `tuple` rather than `int`. The `inputs` of our `rand()` functions are consistent with [`torch.rand()`](https://pytorch.org/docs/stable/generated/torch.rand.html), which has no default values for `size`.